### DISCUSSION TITLE
Multiple standard charge methodologies

### DISCUSSION THREAD

**Discussion by ACLubin** [CMSgov/hospital-price-transparency#64] [Q&A] [answered]
**Scenario: 100% of the Medicare full DRG for all authorized and covered services. Additional days beyond those required to receive 100% of the Medicare full DRG rate will be reimbursed at $___ per diem.**
1) Should both the **standard_charge|negotiated_dollar** and **standard_charge|negotiated_percentage** be encrypted?
2) What **standard_charge|methodology** should be encrypted: the case rate, the per diem rate or both?

**Comment by RidenourK** [comment, marked as answer]
@ACLubin. Thank you for your question. 

Unless otherwise specified, beginning July 1, 2024, each hospital must encode in its machine-readable file all standard charge information, as applicable, for each of the following required data elements…. Whether the standard charge indicated should be interpreted by the user as a dollar amount, or if the standard charge is based on a percentage or algorithm. If the standard charge is based on a percentage or algorithm, the machine-readable file (MRF) must also describe the percentage or algorithm that determines the dollar amount for the item or service, and, beginning January 1, 2025, calculate and encode an estimated allowed amount in dollars for that item or service. Please refer to [45 CFR § 180.50 (b)(2)(ii)(C).](https://www.ecfr.gov/current/title-45/part-180/section-180.50#p-180.50(b)(2)(ii)(C)) 

Please refer to the examples available on this Hospital Price Transparency – Data Dictionary GitHub repository for examples of how to encode data in this scenario.
